x-pack/plugins/index_management/server/config.ts

```diff
@@ -52,6 +52,8 @@ const schemaLatest = schema.object(
       // We take this approach in order to have a central place (serverless.yml) for serverless config across Kibana
       serverless: schema.boolean({ defaultValue: true }),
     }),
+    // Embedded Developer Console is disabled by default
+    enableEmbeddedConsole: schema.boolean({ defaultValue: false }),
```

Comment:
I am intentionally not restricting this to just serverless since we will soon want this enabled for dedicated as well, but there needs to be some other supporting work done for this to work in dedicated so defaulting this to `false` for now.